disclosure_directory/models.py

```diff
@@ -28,6 +30,12 @@ class Meta:
     end_time = models.TimeField(_("End Time"), max_length=255,
                                 null=True, blank=True)
 
+    locations = models.ManyToManyField(Location, blank=True)
+    institutions = models.ManyToManyField(Institution, blank=True)
+
+    is_online = models.BooleanField(default=False)
```

Comment:
No lugar de `is_online`, usar `attendence`, com os valores: 
Live – All attendees are physically present in one location
Virtual – People attend the event entirely online
Hybrid – Some people attend in person, others online